src/main/java/com/uniqueauction/web/user/request/JoinRequest.java

```diff
@@ -28,19 +28,18 @@ public class JoinRequest {
 	@RegExp(regExpCode = PHONE)
 	private String phone;
 
-	public User convert(JoinRequest joinRequest) {
+	public User convert() {
 
 		Role requestRole = Role.CUSTOMER;
 
-		if (joinRequest.getIsAdmin()) {
+		if (this.getIsAdmin()) {
```

Comment:
this를 쓰시려면 직접 사용 할 수 있습니다.
```suggestion
		if (this.IsAdmin) {
```

함수를 사용하시려면 this를 쓰지 않을 수 있습니다
```suggestion
		if (getIsAdmin()) {
```